zipkin-ui/js/component_data/default.js

```diff
@@ -21,7 +21,7 @@ export default component(function DefaultData() {
     const query = convertToApiQuery(window.location.search);
     const serviceName = query.serviceName;
     if (serviceName) {
-      const apiURL = `/api/v1/traces?${queryString.stringify(query)}`;
+      const apiURL = `/zipkin/api/v1/traces?${queryString.stringify(query)}`;
```

Comment:
is this only affecting crossroads? I don't expect us to mount the api under the subpath /zipkin at the server level by default.